src/components/shift-schedule/ScheduleCellContent.tsx

```diff
@@ -0,0 +1,51 @@
+import { TableCell } from '@/components/ui/Table';
+import { cx } from '@/lib/utils';
+import { UserIcon, UsersIcon } from 'lucide-react';
+import { useTranslations } from 'next-intl';
+import type React from 'react';
```

Comment:
This is not needed. Nextjs imports react for types everywhere for us so we dont have to do it.